engines/python/setup/djl_python/transformers_neuronx.py

```diff
@@ -122,10 +119,39 @@ def load_inf2_model(self, model_type, load_path):
             n_positions=self.n_positions,
             unroll=self.unroll)
 
+    def load_inf2_model_from_memory(self, model_type):
```

Comment:
what is making a difference with above one? Just an extra step to load with low memory?

Reply:
does this work as the same as we are passing in `low_cpu_mem_usage=True`

Reply:
It provides an alternative to the model split to disk and load from disk workflow.
It directly transfers the model weights in-memory.

Reply:
More details provided in the CR description